sekoia-io-xdr/list_alerts.py

```diff
@@ -8,13 +8,19 @@
 
 def list_alerts(config, params):
     url: str = OPERATION_CENTER_BASE_URL
+
+    if params.get("start_date") and params.get("end_date"):
```

Comment:
If one of the two parameters is set, we should validate the condition
```suggestion
    if params.get("start_date") or params.get("end_date"):
```